src/lib/components/Swap/SwapButton.tsx

```diff
@@ -35,7 +35,7 @@ function useIsPendingApproval(token?: Token, spender?: string): boolean {
   return Boolean(usePendingApproval(token, spender))
 }
 
-export default function SwapButton({ disabled }: SwapButtonProps) {
+export default memo(function SwapButton({ disabled }: SwapButtonProps) {
   const { account, chainId } = useActiveWeb3React()
```

Comment:
Memoized components will still update if their internal state changes. memo just prevents a re render from props, if it has already rendered with those props. 

It's used to prevent children components from rerendering given the same props, because otherwise functional component renders are triggered by parent renders.